mlir/lib/Conversion/ArithToEmitC/ArithToEmitC.cpp

```diff
@@ -317,14 +317,9 @@ class SelectOpConversion : public OpConversionPattern<arith::SelectOp> {
 // Floating-point to integer conversions.
 template <typename CastOp>
 class FtoICastOpConversion : public OpConversionPattern<CastOp> {
-private:
-  bool floatToIntTruncates;
-
 public:
-  FtoICastOpConversion(const TypeConverter &typeConverter, MLIRContext *context,
-                       bool optionFloatToIntTruncates)
-      : OpConversionPattern<CastOp>(typeConverter, context),
-        floatToIntTruncates(optionFloatToIntTruncates) {}
+  FtoICastOpConversion(const TypeConverter &typeConverter, MLIRContext *context)
+      : OpConversionPattern<CastOp>(typeConverter, context) {}
```

Comment:
```suggestion
  using OpConversionPattern::OpConversionPattern;
```
should work instead (maybe with template args).

Reply:
Sure! Thanks for picking this up :)